design/manual-authn-and-authz/README.md

```diff
@@ -0,0 +1,65 @@
+# Manual Authentication and Authorization workaround
+
+* Status: **Implemented**
+* Discussion: [GitHub PR](https://github.com/strimzi/strimzi-kafka-operator/pull/623)
+
+## Motivation
+
+The long term plan for Strimzi is to have custom resources to define Users for Authentication and Authorization rules.
+The custom resources should be read by operator style application which would be responsible for the setup of the users or ACL rules.
+Until such operator (User Operator - UO) is available, it would be useful to have a way how to setup Authentication and Authorization at least in manual / semi-manual way.
```

Comment:
Until such an operator (User Operator - UO) is available, it would be useful to have a way to setup Authentication and Authorization at least in a manual / semi-manual way.